core/templates/dev/head/css/oppia.css

```diff
@@ -386,6 +377,16 @@ p:last-child {
   font-size: 1em !important;
 }
 
+.oppia-rte ol, .oppia-rte ul {
```

Comment:
You can also add `, .oppia-rte-editor ol, .oppia-rte-editor ul` here and remove the next rule block because the rules are the same

Reply:
Done!